src/factories/docker/push.js

```diff
@@ -0,0 +1,24 @@
+// https://docs.docker.com/engine/reference/commandline/push/
+
+"use strict";
+
+const _ = require('lodash');
+const buildCommand = require('../buildCommand');
+const buildImageName = require('../buildImageName');
+
+const executable = 'docker push'
+
+const optionMap = {
+  disable_content_trust: value => `--disable-content-trust=${value}`
+};
+
+function getTarget(config) {
+  if(config.image == undefined) return '';
+  return buildImageName(config);
```

Comment:
Should we also use buildImageName for the tags on build? The scripts on the TeamCity docker template do tag builds in the project:branch form.